templates/components/_header.html

```diff
@@ -26,7 +26,7 @@
 								<li class="menu-item-has-children">
 									<a href="javascript:void(0);">More</a>
 									<ul class="sub-menu">
-										<li><a href="/#speakers">Speakers</a></li>
+										<li><a href="#speakers">Speakers</a></li>
```

Comment:
@thisisayush firstly I don't think you have run the build.py because the changes is reflected only on index.html whereas running the build.py would have affected all html files including this header. Secondly, the solution as said by @jasonbraganza you can put something like `index.html#speakers`. I think that would work.